src/app/Topology/Shared/utils.tsx

```diff
@@ -41,26 +50,41 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
       : [
           NodeStatus.warning,
           {
-            title: 'Failed to compute JVM ID',
+            title: 'Failed to generate the JVM identifier',
             description: (
               <>
-                <Text component={TextVariants.p}>
-                  If target has JMX Authentication enabled, add the credential to Cryostat keyring.
-                </Text>
-                <Text component={TextVariants.p}>
-                  If the target has SSL enabled over JMX, add its certificate to Cryostat truststore.
-                </Text>
+                <Text component={TextVariants.p}>Check the target authentication settings:</Text>
```

Comment:
add the 'If JMX authentication is enabled, add credentials. If SSL is enabled for JMX, add the SSL certificate.' text?

Reply:
That is there, just a few lines further down: https://github.com/cryostatio/cryostat-web/pull/1119/files/27c2cbc950bb3f0a6e479e64c6d15a120e727e2f#diff-4e598093f82da69d519e2b5a8983e8fd775537388e1cf781f4b3ddb4a5a194adR61